aries-site/src/pages/design-tokens/layout-and-spacing.mdx

```diff
@@ -0,0 +1,93 @@
+import { medium } from 'hpe-design-tokens';
+import { Box, Text } from 'grommet';
+import { Checkmark } from 'grommet-icons';
+import { ContentPane, BestPracticeGroup, Example } from '../../layouts';
+import { DesignTokensTable } from '../../components';
+import { SpacingBestPractices } from '../../examples';
+
+The proximity and grouping of elements establish a clear content hierarchy that facilitates a user in accomplishing their goals, without additional distractions.
+
+Layouts should be:
+
+- **Responsive**: They should adapt to different screen sizes and device types.
+- **Consistent**: They should maintain a predictable structure across the application.
+- **Semantic**: They should be designed with HTML semantics in mind to ensure accessibility.
+
+## Foundations
+
+### Sizing system
+
+The sizing scale consists of a primary scale based on the doubling/halving of a 24px base unit and a supplementary scale doubling/halving a 16px base unit.
+
+This results in the following: 1, 2, 3, 4, 6, 8, 12, 16, 24, 32, 48, 64, 96, 128, 192, 256, 384, 512, 768, 1024, 1536.
```

Comment:
I wonder if there is another way we can display the sizing I feel like its a little harsh to read the way it is, but Im not sure if a table or something would be overkill

Reply:
Good comment -- was weighing the same in terms of taking up least space possible (since people won't be using these numbers directly, it's more an FYI) vs readability/scanability.

Will see what @jcfilben @halocline might think

Reply:
I get that users wont really use the numbers directly so would be fine with leaving as is just didnt know readability wise